### DISCUSSION TITLE
How to render action in a view, custom page

### DISCUSSION THREAD

**Discussion by kvostyc** [filamentphp/filament#11147] [Help]
### Package

Actions

### Package Version

v2.17

### How can we help you?

Hey, im wondering how can i put my action buttons in view blade file?

I have created Custom page with forms etc.

```
<div>
    <x-filament::form wire:submit.prevent="create">
        {{ $this->form }}

     // Somewhere here i want render my actions defined in php class
    </x-filament::form>
</div>
```
```
protected function getFormActions(): array
    {
        return [
            Action::make('settings')->action('openSettingsModal'),
        ];
    }
}
```

But i can´t find it in documentation, im desperate rn :D

Can someone help me with that or link me to documentation ? If there is any about this problem.

Thanks

- Kvostyc

**Comment by leandrocfe** [comment]
https://filamentphp.com/docs/3.x/actions/adding-an-action-to-a-livewire-component#adding-the-action

**Reply by kvostyc** [reply]
> https://filamentphp.com/docs/3.x/actions/adding-an-action-to-a-livewire-component#adding-the-action

Thanks, but im using filament 2.17 and i cant upgrade it because of other dependencies.

**Reply by leandrocfe** [reply]
Hum, sorry..
